### DISCUSSION TITLE
Sans Dodge Mechanic

### DISCUSSION THREAD

**Discussion by DaveFan19** [Pico8430/Indie-Cross-PE-Port#7] [Show and tell]
https://user-images.githubusercontent.com/99294235/164910954-30e9e546-a898-4a53-98ae-0d647b4d7d5c.mp4

i'll make a pull request when i get my version of the port updated if you want

**Comment by Pico8430** [comment]
WHY DOES THIS LOOKS SO GOOD AUUUGH

**Comment by DaveFan19** [comment]
just pull requested
